apps/services/temporary-voter-registry-api/src/app/modules/voterRegistry/voterRegistry.controller.ts

```diff
@@ -33,9 +33,10 @@ export class VoterRegistryController {
   }
 
   @ApiOkResponse({
-    description: 'Finds voter registry entry given user authentication',
+    description: 'Finds voter registry entry given users national id',
     type: VoterRegistry,
   })
+  @BypassAuth() // currently we have no way to create system auth tokens, we will add those when available for better tracking
```

Comment:
Hm? This should be possible, by using client credentials to get an access token with some endorsement client and system scope. 

Please follow this up with a PR to fix this.

Reply:
Hmm, I might not familiar enough with the auth library 😅 
This is initialized via cron job that uses the metadata service to get the data.
Are we able to create a system token without user credentials? Though this was not yet implemented?

Reply:
Yes, this is built in functionality in Identity Server. I'm not sure if we have any reference code or helpers yet, but Fuglar can provide implementation details.

Reply:
Nice, will contact them about this ❤️

Reply:
Thank you ❤️